<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>657</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QComboBox" name="method">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>110</y>
      <width>191</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>BWT and RLE and Huffman</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>BWT</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>RLE</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Huffman</string>
     </property>
    </item>
   </widget>
   <widget class="QTextEdit" name="input">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>30</y>
      <width>351</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="output">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>460</y>
      <width>381</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="process_button">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>100</y>
      <width>75</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Process</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>170</y>
      <width>571</width>
      <height>271</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
   </widget>
   <widget class="QComboBox" name="operation">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>110</y>
      <width>121</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Encode</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Decode</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Encode â†’ Decode</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="open_file_button">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>30</y>
      <width>75</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>open file</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>657</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
